n2y/blocks.py

```diff
@@ -63,7 +63,6 @@ def to_pandoc(self):
         raise NotImplementedError()
 
     def children_to_pandoc(self):
-        assert self.has_children
```

Comment:
Is this intentional? If so, why are we removing it?

Reply:
yes, because if it's called on a block with no children, we want it to return an empty list instead of throwing an error.